Parse/PFQuery.m

```diff
@@ -635,7 +635,7 @@ + (instancetype)queryWithClassName:(NSString *)className normalizedPredicate:(NS
 ///--------------------------------------
 
 - (void)checkIfCommandIsRunning {
-    @synchronized (self) {
+    @synchronized(self) {
```

Comment:
The proper one - no space, same as like `@weakify()` or `@strongify`, as this is not a control statement. At least - according to clang-format.